### DISCUSSION TITLE
Having issues connecting 1 Controller to 4 Gamecubes.

### DISCUSSION THREAD

**Discussion by LEOsMIND** [darthcloud/BlueRetro#707] [Q&A] [answered]
My goal is to control 4 Gamecubes with 1 Controller at the same time.
I managed to duplicate the mapping of the first device thanks to this post https://github.com/darthcloud/BlueRetro/discussions/675

Now I can control all 4 outputs with one controller, but unfortunately only if 4 controllers are connected.
Can I somehow bypass that.

Thank you.

**Comment by darthcloud** [comment, marked as answer]
ok that's a problem caused by the port state being linked directly to bt connection state rather than the adapter state.

I will have to rework that for next version.

For now i think you got to have 4 controller connected unfortunately.

**Comment by darthcloud** [comment]
I fixed this in the v1.9 beta:
https://github.com/darthcloud/BlueRetro/actions/runs/6413711330

All the port that one controller map will be enable now.
